runtimes/eden/src/lib.rs

```diff
@@ -82,6 +82,7 @@ construct_runtime! {
         RandomnessCollectiveFlip: pallet_randomness_collective_flip = 4,
 
         // Governance
+        Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 9,
```

Comment:
I know I am a pain but `Preimage` probably does not fit in the `Governance` category (which also starts at `10`, not `9`). I'd suggest moving it to `System` or `Neat Things` depending on which category best fits it